project/project.go

```diff
@@ -337,6 +383,16 @@ func (p *Project) Down(ctx context.Context, opts options.Down, services ...strin
 		return err
 	}
 
+	if opts.RemoveVolume {
```

Comment:
Ok, checked, `rm -v` does not remove declared/named volumes, so it's working the same :)